addons/AI_Assistant/src/presenter.js

```diff
@@ -0,0 +1,1556 @@
+function AddonAI_Assistant_create() {
+
+    var presenter = function () {};
+
+    presenter.setPlayerController = function (controller) {
+        presenter.playerController = controller;
+        presenter.eventBus = controller.getEventBus();
+        presenter.eventBus.addEventListener('ValueChanged', this);
+        presenter.eventBus.addEventListener('PageLoaded', this);
+        presenter.commander = controller.getCommands();
+    };
+
+    presenter.fetchSessionJWTToken = function() {
+        presenter.isFetching.request = true;
+        let result = fetch('/api/v2/jwt/session_token', {
+            method: 'GET'
+        });
+        return result;
+    };
+
+    let errorMessages = {
```

Comment:
`const`, nie `let`. Przy okazji warto zastsować UPPER_SNAKE_CASE (Screaming Snake Case), gdyż te wartości są znane i nigdy się nie zmienią. Warto stosować UPPER_SNAKE_CASE, gdyż w js jeżeli nada się const to dalej można zmienić wartość w objekcie (np. zmienić wartość dla transcript). Zastosowanie takiego zapisu przynajmniej dla programisty daje dodatkowe informacje